pubsub/README.md

```diff
@@ -0,0 +1,58 @@
+# Google Cloud Pub/Sub Kotlin Sample
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googlecloudplatform/kotlin-samples&page=editor&working_dir=pubsub
+
+## Description
+
+This simple Kotlin command-line application demonstrates how to access the Pub/Sub API using
+the [Google Cloud Client Library for Java][google-cloud-java].
+
+[Google Cloud Pub/Sub][pubsub] is a fully-managed real-time messaging service that allows you to
+send and receive messages between independent applications.
+
+[pubsub]: https://cloud.google.com/pubsub/
+[google-cloud-java]: https://github.com/GoogleCloudPlatform/google-cloud-java
+
+## Quickstart
+
+#### Setup
+- Install [Maven](http://maven.apache.org/).
+- [Enable](https://console.cloud.google.com/apis/api/pubsub.googleapis.com/overview) Pub/Sub API.
+- Set up [authentication](https://cloud.google.com/docs/authentication/getting-started).
+
+#### Build
+- Build your project with:
+```
+  mvn clean package -DskipTests
```

Comment:
our other samples use Gradle. Let's look into having both included in this sample.

Reply:
Since other samples are using Gradle, it's not fair to have both for this one. I will try to convert the project into Gradle.

Reply:
If possible though, it would be good to have both for all samples. So I'd rather you not remove your hard work!